### PR TITLE
Fixed link in offering-HIPAA-HITECH.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/Compliance/issues/51.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

File: compliance/regulatory/offering-HIPAA-HITECH.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ The [Service Trust Portal](https://www.microsoft.com/trustcenter/STP/default.asp
 
 To assist customers with this task, Microsoft has published these guides:
 
-- *HIPAA/HITECH Act implementation guidance* for [Azure](/azure/governance/blueprints/samples/hipaa-hitrust/) and for [Dynamics 365 and Office 365](https://go.microsoft.com/fwlink/?LinkID=257510). Written for privacy, security, and compliance officers and others responsible for HIPAA and HITECH Act implementation, they describe concrete steps your organization can take to maintain compliance.
+- *HIPAA/HITECH Act implementation guidance* for [Azure](/azure/governance/blueprints/samples/hipaa-hitrust-9-2/) and for [Dynamics 365 and Office 365](https://go.microsoft.com/fwlink/?LinkID=257510). Written for privacy, security, and compliance officers and others responsible for HIPAA and HITECH Act implementation, they describe concrete steps your organization can take to maintain compliance.
```
Comment (JohanFreelancer9) [inline compliance/regulatory/offering-HIPAA-HITECH.md:98]:
```suggestion
- *HIPAA/HITECH Act implementation guidance* for [Azure](/azure/governance/blueprints/samples/hipaa-hitrust-9-2/) and for [Dynamics 365 and Office 365](https://go.microsoft.com/fwlink/?LinkID=257510). Written for privacy, security, and compliance officers and others responsible for HIPAA and HITECH Act implementation, they describe concrete steps your organization can take to maintain compliance.
```

**Pull request merged**